src/app/components/FooterComponent.js

```diff
@@ -85,7 +85,7 @@ const FooterComponent = (props) => {
             <Col>
               <p style={{ fontSize: '.8em' }}>
                 {`Copyright © ${new Date().getFullYear()} `}
-                RSK Labs. All rights reserved.
+                RootstockLabs. All rights reserved.
```

Comment:
@LordCheta  Not sure if it should be with or without space. I'd probably go without.
```suggestion
                Rootstock Labs. All rights reserved.
```